common/src/main/java/com/wynntils/gui/screens/settings/WynntilsBookSettingsScreen.java

```diff
@@ -30,16 +37,21 @@
 import net.minecraft.client.gui.components.events.GuiEventListener;
 import net.minecraft.client.gui.screens.Screen;
 import net.minecraft.network.chat.TranslatableComponent;
+import org.lwjgl.glfw.GLFW;
 
 public class WynntilsBookSettingsScreen extends Screen implements TextboxScreen {
     private final int FEATURES_PER_PAGE = 13;
+    private final int CONFIGS_PER_PAGE = 3;
     private final List<AbstractButton> features = new ArrayList<>();
+    private final List<AbstractButton> configs = new ArrayList<>();
 
     private TextInputBoxWidget focusedTextInput;
     private final SearchWidget searchWidget;
     private ScrollButton featureListScrollButton;
+    private ScrollButton configListScrollButton;
     private Feature selected = null;
-    private int scrollOffset = 0;
+    private int featureScrollOffset = 0;
+    private int configScrollOffset = 0;
 
```

Comment:
A decent bit of code duplication, though likely can not be fixed

Reply:
Yeah...